examples/nextjs/pages/api/graphql.ts

```diff
@@ -1,34 +1,20 @@
-import {
-  getGraphQLParameters,
-  processRequest,
-  renderGraphiQL,
-  sendResult,
-  shouldRenderGraphiQL,
-} from 'graphql-helix';
-import { NextApiHandler } from 'next/types';
-import { schema } from '../../graphql/schema';
-
-export default (async (req, res) => {
-  const request = {
-    body: req.body as object,
-    headers: req.headers,
-    method: req.method!,
-    query: req.query,
-  };
+/** GraphQL API entrypoint */
 
-  if (shouldRenderGraphiQL(request)) {
-    res.send(renderGraphiQL({ endpoint: '/api/graphql' }));
-  } else {
-    const { operationName, query, variables } = getGraphQLParameters(request);
+import { createYoga } from 'graphql-yoga';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { schema } from '../../graphql/schema';
 
-    const result = await processRequest({
-      operationName,
-      query,
-      variables,
-      request,
-      schema,
-    });
+export const config = {
+  api: {
+    // Disable body parsing (required for file uploads)
+    bodyParser: false,
+  },
+};
 
-    void sendResult(result, res);
-  }
-}) as NextApiHandler;
+export default createYoga<{
+  req: NextApiRequest;
+  res: NextApiResponse;
+}>({
+  schema,
```

Comment:
I know not all existing examples have this, but all examples should have a context factory that at a minimum creates an empty context object.  Some Pothos plugins require a context object, and it's good practice to set this up for any new graphql api

Reply:
I agree and I'll add a context factory